README.md

```diff
@@ -19,58 +19,82 @@ There is always ongoing work being made to the testnet - we have made this trans
 
 - https://github.com/orgs/paseo-network/projects/1/views/3
 
-## Large changes to the network
+## Impactful changes to the network and its operation
 Changes to the network are covered by raising a PAS (Paseo Action Submission). Similar to EIPs these are to document changes to how the network will progress.
-Anyone is able to raise a PAS for review by the maintainers of the network. 
-The raised PASs can be found at: 
+Anyone is able to raise a PAS for review by the maintainers of the network.
+The raised PASs can be found at:
 
 - https://github.com/paseo-network/paseo-action-submission
 
 ## $PAS the testnet token
-All actions made on the relay chain, will need to be made in $PAS 
+All actions made on the relay chain, will need to be made in $PAS
 
 ## Getting the testnet token ($PAS)
-1. Faucet -> https://paritytech.github.io/polkadot-testnet-faucet/
+1. Faucet -> https://faucet.polkadot.io/
 2. Guide on how to interact with the faucet -> https://medium.com/p/8c2fbe45b603
 
 ## Adding your tokens to the faucet
 Are you a team deploying on Paseo that wants to make their tokens availabe on the fuacet for builders? (TODO)
 
+## Want to connect a chain to Paseo?
+Please read -> [PAS-10: Onboard chains via coretime](https://github.com/paseo-network/paseo-action-submission/blob/main/pas/PAS-10-Onboard-paras-coretime.md).
+Or open an issue at: https://github.com/paseo-network/support
+
 ## Want to become a validator on Paseo?
-Please read -> https://github.com/paseo-network/paseo-action-submission/blob/main/pas/onboard_infrastructure_providers.md
+Please read -> [PAS-1: Onboard infrastructure providers](https://github.com/paseo-network/paseo-action-submission/blob/main/pas/PAS_ID1_onboard_infrastructure_providers.md).
```

Comment:
I would add a comment maybe that for the mooment the applications are closed as we are not aiming to onboard any new provider for now.

Reply:
Maybe not the best place to keep the status of this, but I'll include it